src/lib/Section/Section.svelte

```diff
@@ -19,25 +19,27 @@
 
 <style>
 	.section {
-		padding-top: 2em;
-		padding-left: 2em;
+		padding-top: 2rem;
+		padding-left: 2rem;
+		padding-right: 2rem;
 	}
 
 	.section h3 {
 		font-family: 'Bebas Neue';
 		font-style: normal;
 		font-weight: 400;
-		font-size: 48px;
-		line-height: 58px;
-		margin-bottom: 0px;
-		margin-top: 0px;
+		font-size: 3rem;
+		line-height: 3.625rem;
+		margin-bottom: 0rem;
+		margin-top: 0rem;
 	}
 
 	.section hr {
-		width: 570px;
-		margin-left: 0px;
-		margin-top: 20px;
-		height: 0.35px;
+		width: 35.625rem;
+		margin-left: 0rem;
+		margin-top: 1.25rem;
+		height: 0.021875rem;
```

Comment:
So does this - maybe it would be better to use fractions where applicable?

Reply:
I agree that using fractions when the decimal is too long would make the code more readable. How about this SRS?
* All CSS values MUST use fractions when the length of the floating point decimal is greater than two.

Reply:
I don't know a rule in eslint that enforces that, and I would prefer to avoid manual style guides.